src/Zbot/Service/GitHub.hs

```diff
@@ -139,15 +154,143 @@ closeIssue project ctx number accessToken = do
 
             errorMsg = "Encountered an error while closing the issue."
 
+addTnakChatEmoji :: T.Text -> T.Text -> GitHubAction
+addTnakChatEmoji emoji url ctx options
```

Comment:
i wonder if we should just downcase all characters as well

Reply:
That's a good idea, updated.